mlir/lib/Transforms/RemoveDeadValues.cpp

```diff
@@ -617,23 +736,34 @@ void RemoveDeadValues::runOnOperation() {
   auto &la = getAnalysis<RunLivenessAnalysis>();
   Operation *module = getOperation();
 
+  // Tracks values eligible for erasure - complements liveness analysis to
+  // identify "droppable" values.
+  DenseSet<Value> deadVals;
+
+  // Maintains a list of Ops, values, branches, etc., slated for cleanup at the
+  // end of this pass.
+  RDVFinalCleanupList finalRDVFinalCleanupList;
```

Comment:
[nit] "final" used twice in this variable name 😅

Reply:
done, thank you!